oshc/main/templates/contest_submission.html

```diff
@@ -4,10 +4,10 @@
 
 <div class="jumbotron text-xs-center">
   <div class="container text-center">
-      <h1 class="display-3">Thank You!</h1>
-      <p class="lead"><strong>Your contest has been submitted for review.</strong></p>
+      <h1 class="display-3 text-center">Thank you!</h1>
+      <p class="lead text-center"><strong>Your contest has been submitted for review.</strong></p>
```

Comment:
This isn't working on my local or on a trial deployment. Can you please share screenshots?

Reply:
Sure ma'am, here's the screenshot of my local deployment.
![local_oshc](https://user-images.githubusercontent.com/4662916/31834029-266e0456-b5ea-11e7-915f-28688709a4bd.png)

Reply:
If you check this is what the current status is. 

I added few more words in the description of the issue. By center I meant, vertically and horizontally in the center of the page, somewhere in the middle.

Reply:
@tapasweni-pathak it's done, have a look.
![done_oshc](https://user-images.githubusercontent.com/4662916/31835796-98c8a456-b5f0-11e7-8229-28318fd6fab4.png)